manager/kubernetes.go

```diff
@@ -22,7 +22,7 @@ var (
 )
 
 const (
-	KubeStatusPollConut    = 5
+	KubeStatusPollConut    = 30
```

Comment:
Typo.

Also, this is a BLOCKING API call. You won't expect the caller of API waiting for 30 seconds for the update. Need to do it in another way.